sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/SearchOptions.java

```diff
@@ -19,42 +19,42 @@
      * Default is false. Setting this value to true may have a performance
      * impact. Note that the count returned is an approximation.
      */
-    @JsonProperty(value = "")
+    @JsonProperty(value = "includeTotalCount")
     private Boolean includeTotalCount;
 
     /*
      * The list of facet expressions to apply to the search query. Each facet
      * expression contains a field name, optionally followed by a
      * comma-separated list of name:value pairs.
      */
-    @JsonProperty(value = "")
+    @JsonProperty(value = "Facets")
```

Comment:
is this shoud be lower-case for the  first letter

Reply:
Hmm, good catch on this, the JSON name is `facets` so I'm not certain why this is being generated with capitalization unless this property in copied over to another value which is using the correct JSON naming. I'll follow-up on the Swagger for this.

Reply:
These values are passed into another method where these annotations aren't applied.